src/app/snapshot/snapshots-page/snapshot-filter/snapshot-filter.container.ts

```diff
@@ -37,13 +36,15 @@ const FILTER_KEY = 'snapshotFilters';
       [firstDayOfWeek]="firstDayOfWeek$ | async"
       [selectedAccounts]="(filters$ | async).accounts"
       [selectedVms]="(filters$ | async).vmIds"
```

Comment:
You have `selectedVms` and `selectedVmIds` in your inputs. It may look like selectedVms means vm objects and selectedVmIds means vm ids, but you pass vmIds to the first input and volumeVmIds to the second input. This is confusing.